src/shared/Microsoft.Git.CredentialManager/Authentication/MicrosoftAuthentication.cs

```diff
@@ -205,6 +234,15 @@ private async Task<IPublicClientApplication> CreatePublicClientApplicationAsync(
                 appBuilder.WithParentActivityOrWindow(() => new IntPtr(hWndInt));
             }
 
+            // On Windows 10 & .NET Framework try and use the WAM broker
+            if (enableBroker && PlatformUtils.IsWindows10())
```

Comment:
Broker is also available on Win Server 2016 and Win Server 2019, but the support there is a little iffy.

Reply:
NT major version 10 should also capture Windows Server SKUs based on Windows 10.